network/unstaked/conduit.go

```diff
@@ -0,0 +1,26 @@
+package unstaked
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/network"
+)
+
+// UnstakedConduit is a special conduit used by unstaked nodes which wraps
+// the default conduit implementation and replaces the target of every network
+// send with the upstream staked node.
+type UnstakedConduit struct {
```

Comment:
nit: I would call this `ProxyConduit` or `FixedConduit` instead of unstaked conduit to describe what it does instead of where it is being used.

Reply:
this is a good point.